### PR TITLE
Lecture 04-a

### PR DESCRIPTION
**Pull request by ygrepo**
In this class we learn more about  Convolution Networks by studying striding and convolution “a trou”. We also cover concrete applications of ConvNets in object and face detection. One important application for ConvNets is semantic segmentation which was the major technology used in the self-driving car DARPA’s LAGR program  and is also largely present in medical imaging for tumor detection. We finally go over the most recent state of the art Convolution Neural Network architectures.

-- First part of Lecture 4 (25 Feb 2019) by Pedro Manuel Herrero Vidal and Doruk Kilitcioglu. Edited by Yves Greatti

### CODE REVIEW

**Comment by Atcold** [comment]
PR summary, author list, and date are missing.
Plz, edit the first message, similarly to how it's done in #1.

**Comment by ygrepo** [comment]
> PR summary, author list, and date are missing.
> Plz, edit the first message, similarly to how it's done in #1.

See above, let me know if this is good.
Yves

**Comment by Atcold** [comment]
Very well, thanks.

**Comment by Atcold** [comment]
Buhahahaha! 🤣 
screenshot 2019-03-04 14 57 28

File: lectures/04-a/theory.tex [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Having a 2-unit layer (same as input layer) right before the output layer allows to visualize the transformations 
+that the original space went throw to solve the classification task. 
+The last linear transformation involves the multiplication of $A^{(3)}$ and $W^{(2)}$, 
+where the different rows of $W^{(2)}$ are the 2D vectors that will point at the different classes in the 2D space. 
+In this representation, the different classes are linearly separable.
+\end{itemize}
+
+\begin{figure}[!h]
+\centering
+\includegraphics[width=170mm]{lectures/04-a/images/ArchitectureForClassificationAndVisualizationInTheInputSpace.png}
```
Comment (Atcold) [inline lectures/04-a/theory.tex]: Hahaha 🤣

Comment (Atcold) [reply lectures/04-a/theory.tex]: I love the diagrams, though! Very well drawn!

**Pull request merged**